include/aspect/simulator.h

```diff
@@ -978,7 +1026,7 @@ namespace aspect
        * This function is implemented in
        * <code>source/simulator/helper_functions.cc</code>.
        */
-      std::pair<double,bool> compute_time_step () const;
+      double compute_time_step () const;
```

Comment:
You'll have to update/delete the comment describing what the second part of the returned `std::pair` is.